apps/dav/lib/Connector/Sabre/Directory.php

```
@@ -250,6 +256,9 @@ public function getChildren() {
 			return $this->dirContent;
 		}
 		try {
+			if (!$this->info->isReadable()) {
+				throw new Forbidden('No read permissions');
```

throwing `NotFound` above and `Forbidden` here seems weird to me. If the condition is the same they should throw the same exception.